sources/Interop/Windows/winrt/wrl/client/ComPtr`1.Manual.cs

```diff
@@ -4,150 +4,401 @@
 // Original source is Copyright © Microsoft. All rights reserved.
 
 using System;
+using System.ComponentModel;
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 using static TerraFX.Interop.Windows;
 
 namespace TerraFX.Interop
 {
-    public unsafe partial struct ComPtr<T> : IDisposable
+    /// <summary>
+    /// A type that allows working with pointers to COM objects more securely.
+    /// </summary>
+    /// <typeparam name="T">The type to wrap in the current <see cref="ComPtr{T}"/> instance.</typeparam>
+    /// <remarks>
+    /// While this type is not marked as <see langword="ref"/> so that it can also be used in fields, make sure
+    /// to keep the reference counts properly tracked if you do store <see cref="ComPtr{T}"/> instances on the heap.
+    /// </remarks>
+    public unsafe struct ComPtr<T> : IDisposable
         where T : unmanaged
     {
+        /// <summary>
+        /// The raw pointer to a COM object, if existing.
+        /// </summary>
         private T* ptr_;
 
+        /// <summary>
+        /// Creates a new <see cref="ComPtr{T}"/> instance from a raw pointer and increments the ref count.
+        /// </summary>
+        /// <param name="other">The raw pointer to wrap.</param>
         public ComPtr(T* other)
         {
             ptr_ = other;
+
```

Comment:
Unnecessary newline was inserted here and a few other places in the file.
```suggestion
```

Reply:
But they looked so nice 😛
Fixed in b2a4be644512cb4da79f1df6c2c796a294b47b92!